modules/core/src/main/java/org/apache/synapse/rest/RESTUtils.java

```diff
@@ -88,6 +92,54 @@ public static String getFullRequestPath(MessageContext synCtx) {
         return url;
     }
 
+    /**
+     * Populate Message context properties for the query parameters extracted from the url
+     *
+     * @param synCtx MessageContext of the request
+     */
+    public static void populateQueryParamsToMessageContext(MessageContext synCtx) {
+
+        String path = RESTUtils.getFullRequestPath(synCtx);
```

Comment:
We can just call the method here.

  ```suggestion
        String path = RESTUtils.getFullRequestPath(synCtx);
```